discussion-topics/topics.md

```diff
@@ -18,23 +18,24 @@ The community support squad memebers works on different day to day tasks and it
 - A community post or GitHub issue which needs to be discussed together
 - Someone new you have learned (e.g. a tool, script, command etc.) and want to share the knowledge with the team.
 
-**Types of issues to discuss and partner on:** 
-- Tough issues where a user and support engineer could benefit from a developer’s knowledge to solve and learn
-- Highlight interesting use cases, bugs, themes, or feature requests
-
-**Meeting format:**
-- 30 to 60 minutes weekly
+**Meeting/Workshop format:**
+- 30 to 60 minutes weekly (or bi-weekly)
 - Complete the table below to provide the list of topics you want to bring for discussion
-- We'll post the agenda two days in advance here in github
+- Create a small presentation document to present your topic (notihng fancy)
+
+
+## Recording of Session 
+These meeting/workshop sessions will be recorded so that can be used by the squad for reference and view later if not able to join it.
```

Comment:
`These meeting/workshop sessions will be recorded so that can be used by the squad for reference and view later if not able to join it.` -> These sessions will be recorded so they can be used by the squad for reference and view later if not able to join it.